src/tools/rbd/Utils.cc

```diff
@@ -730,7 +730,7 @@ void init_context() {
 }
 
 int init(const std::string &pool_name, librados::Rados *rados,
-         librados::IoCtx *io_ctx) {
+         librados::IoCtx *io_ctx, bool force) {
```

Comment:
Like @dillaman said, this should also be a more descriptive name... bool force_full_try probably